packages/ses/NEWS.md

```diff
@@ -1,5 +1,27 @@
 User-visible changes in SES:
 
+# Next release
+
+- When running Ava tests on Node written in TypeScript, the SES error taming
```

Comment:
There's no need to mention Ava here. This should be for all TypeScript on Node on v8.

Reply:
Done